### PR TITLE
chore(datepickers): update prop descriptions

### PR DESCRIPTION
**Pull request by Mikaelia**
## Description

Updates datepickers package prop descriptions

### CODE REVIEW

File: packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx [hunk 1]
```diff
@@ -14,37 +14,49 @@ import End from './components/End';
...
   /**
-   * Maximum value. Users are still able to manually enter a date above this value.
+   * Disables dates after this value on the calendar element
    */
   maxValue?: Date;
   /**
-   * Returns the parsed start and end dates
+   * Handles start and end date changes
+   *
+   * @param {Object} values The selected start and end dates
```
Comment (jzempel) [inline packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx:41]: I could go with this, for sure. But I wonder if we should use the more thorough syntax like:

```
@param {Object} values The selected dates
@param {Date} [values.startValue] Start date
@param {Date} [values.endValue] End date
```

Or maybe ~The start date~ -> "Optional start date"?

**Review by jzempel — CHANGES_REQUESTED** [review]

File: packages/datepickers/src/elements/Datepicker/Datepicker.tsx [hunk 2]
```diff
@@ -31,63 +31,72 @@ import { StyledMenu, StyledMenuWrapper } from '../../styled';
...
   formatDate?: (date: Date) => string;
   /**
-   * Accepts [all valid Intl locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation)
+   * Applies locale-based formatting.
+   * Accepts [all valid Intl locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation).
    */
   locale?: string;
   /**
-   * Minimum value. Users are still able to manually enter a date below this value.
+   * Disables dates before this value on the calendar element
```
[the remark this answers is not in the document]
Comment (Mikaelia) [reply packages/datepickers/src/elements/Datepicker/Datepicker.tsx:56]: Sure, I think component might be more precise. Another thought is that I could also remove the last word so it is just calendar

File: packages/datepickers/src/elements/Datepicker/Datepicker.tsx [hunk 3]
```diff
@@ -31,63 +31,72 @@ import { StyledMenu, StyledMenuWrapper } from '../../styled';
...
   /**
-   * Show compact styling
+   * Applies compact styling
    */
   isCompact?: boolean;
   /**
-   * Override default date parsing. Receives a localized input value and returns a `Date` object.
+   * Overrides default date parsing
+   *
+   * @param {string} inputValue
```
[the remark this answers is not in the document]
Comment (Mikaelia) [reply packages/datepickers/src/elements/Datepicker/Datepicker.tsx:70]: Oops!

File: packages/datepickers/src/elements/Datepicker/Datepicker.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jzempel) [reply packages/datepickers/src/elements/Datepicker/Datepicker.tsx:56]: Yeah, I like the simplicity of removing altogether

File: packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx (hunk 1 above)
Comment (Mikaelia) [reply packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx:41]: Can do. "Optional end date" as well?

Comment (jzempel) [reply packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx]: Whatever reads best to you.

**Review by jzempel — APPROVED** [review]
Nice tune-up here. Thanks @Mikaelia.

**Review by austingreendev — APPROVED** [review]

**Pull request merged**